### PR TITLE
Avoid calling EntityId::newFromNumber when not needed

### PR DESCRIPTION
**Pull request by thiemowmde**

### CODE REVIEW

**Comment by Ladsgroup** [comment]
:+1:

File: tests/unit/Statement/StatementListTest.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ private function getStatement( $propertyId, $guid, $rank = Statement::RANK_NORMA
 
 		$statement->expects( $this->any() )
 			->method( 'getPropertyId' )
-			->will( $this->returnValue( PropertyId::newFromNumber( $propertyId ) ) );
+			->will( $this->returnValue( new PropertyId( 'P' . $propertyId ) ) );
```
Comment (thiemowmde) [inline tests/unit/Statement/StatementListTest.php]: In production code I would agree, but this is a test. All numbers used here are integer constants from test providers in this file, and never user input.

Comment (JeroenDeDauw) [inline tests/unit/Statement/StatementListTest.php]: What difference does it make if this is user input or not?

Comment (thiemowmde) [inline tests/unit/Statement/StatementListTest.php]: Because I can now do this: #699. I can't do this when it's user input.

**Pull request merged**